apps/server/src/modules/lernroom/usecase/dashboard.uc.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { DashboardEntity } from '@src/entities/learnroom/dashboard.entity';
+import { IDashboardRepo } from '@src/repositories/learnroom/dashboard.repo';
+
+@Injectable()
+export class DashboardUc {
+	constructor(@Inject('DASHBOARD_REPO') private readonly dashboardRepo: IDashboardRepo) {}
```

Comment:
why not using default constructor based DI for the dashboardRepo? what are the other different things that could be injected here instead?